docs/idl_vs_python/python_vs_IDL_fit.rst

```diff
@@ -0,0 +1,48 @@
+###############################
+IDL vs. Python Spectrum Fitting
+###############################
+
+Fit Results
+===========
+
+The surface brightness data in the following spectra are from a southern region of 
+NGC 2023. To learn more about the region of NGC 2023 examined, refer to *The PAH 
+Emission Characteristics of the Reflection Nebula NGC 2023* by Els Peeters et al.
+
+Changes to Default scipack File Used for Fitting
+================================================
+
+Several changes were made to the default package *scipack_ExGal_SpitzerIRSSLLL.ipac*
+when fitting the infrared spectrum of NGC 2023.  Of note, the **amp** values of 
+H2 S(4), H2 S(6) and H2 S(7) were changed to **0**.  Additionally, the **amp_fixed** 
+values of the same features were changed to **True**.
+
+Python vs. IDL PAHFIT Results
+=============================
+
+The graphs below compare the fit results of given pixels of NGC 2023.  The top graph
+shows the Python and IDL spectrum fit along with the data points used.  The second
+graph shows the residuals resulting from the fits.  Lastly, the third graph compares
+the calculated fluxes of PAH features obtained from the fits.  In both graphs, the 
+resulting fit came from default maxiter values (the IDL version of PAHFIT uses a 
+default maxiter of 200 while the Python version uses a default maxiter of 1000).
+
+.. image:: NGC2023_pixel_18_14.png
+
+The above graphs of pixel [18, 14] represent a 'good' comparison between the IDL
+and Python fit.  That is, the fits resulted in similar residuals and calculated 
+fluxes.  Of note, the Python fit does not contain a dust feature at 5.27 µm
+while the IDL fit does. Additionally, both the Python and IDL fit do not contain a 
```

Comment:
The statement that the python fit does not contain the dust feature at 5.27 micron.  Does this mean that the python scipack didn't include the feature?  Or that the python fit set the feature strength to 0?  It would be good to clarify this.  Same comment about the 14.04 feature.